app/components/competency-title-editor.js

```diff
@@ -37,24 +35,20 @@ export default Component.extend(Validations, ValidationErrorDisplay, {
       if (isPresent(competency)) {
         this.set('title', competency.get('title'));
       }
-    }
-  },
+    },
+
+    save() {
+      this.send('addErrorDisplayFor', 'title');
+
+      if (this.validations.isValid) {
```

Comment:
I think you need to call `await this.validate();` before checking `this.validations.isValid`.

Reply:
Isn't that the old way? It should be a synchronous check now, no?

http://offirgolan.github.io/ember-cp-validations/docs/modules/Accessing%20Validations.html

Reply:
While isValid is not async the validations themselves must be run. If a field hasn't been touched they may not have been. I think that's where `this.validate()` comes into play.